src/svm.rs

```diff
@@ -35,6 +35,8 @@ struct SvmRawVec<'a, T> {
     fine_grain_buffer: bool,
 }
 
+unsafe impl<'a, T: Send> Send for SvmRawVec<'a, T> {}
```

Comment:
This is component of `SvmVec`,  but `SvmVec` is not `Send`, is this correct?

Reply:
`SvmVec` is automatically `Send`. If a type consists only of types that are `Send`, then that type is automatically `Send`. This is the reason why the [Interoperability API Guidelines](https://rust-lang.github.io/api-guidelines/interoperability.html#types-are-send-and-sync-where-possible-c-send-sync) say that types should be `Send` where possible. And "where possible" is about "thread safety characteristics", and I think all the types here are safe in that regard.